packages/velaux-ui/src/components/UISchema/index.less

```diff
@@ -5,6 +5,7 @@
       width: 100%;
     }
     .next-form-item-label {
+      text-align:left;
```

Comment:
```suggestion
      text-align: center;
```

Aligning to left make it too left compared to the labels above

Reply:
I have removed this change，and the UI interface is as follows：
![image](https://github.com/kubevela/velaux/assets/55052467/28631355-bbd9-4a1d-94db-bbd91e667961)